content/pt/docs/tutorials/kubernetes-basics/explore/explore-interactive.html

```diff
@@ -0,0 +1,41 @@
+---
+title: Tutorial Interativo - Explorando seu aplicativo
+weight: 20
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+<link href="/docs/tutorials/kubernetes-basics/public/css/overrides.css" rel="stylesheet">
+<script src="https://katacoda.com/embed.js"></script>
+
+<div class="layout" id="top">
+
+    <main class="content katacoda-content">
+
+        <br>
+        <div class="katacoda">
+
+            <div class="katacoda__alert">
+                Para interagir com o Terminal, por favor, use a versão para desktop ou table.
+            </div>
+
+            <div class="katacoda__box" id="inline-terminal-1"  data-katacoda-id="kubernetes-bootcamp/4" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/expose/expose-intro/" role="button">Continue para o Módulo 4<span class="btn__next">›</span></a>
```

Comment:
Hi Jailton, here need include "/pt/" into the path to not break the URL.

Reply:
Hi @jcjesus! There is no **Expose Your App Publicly** translation to Portuguese. I'll translate this section next week.

Reply:
mmmm, ok! Thanks!